test/services/GooglePlacesService.test.ts

```diff
@@ -68,6 +68,17 @@ describe('Unit tests for GooglePlacesService', function () {
       });
   });
 
+  test('Get details for a place using invalid placeID, should be unsuccessful', () => {
+    return googlePlacesAPI
+      .getRestaurantDetails('ChIJ2fF6zviX9YgRflKEZNrpLrQ')
```

Comment:
Could we turn `ChIJ2fF6zviX9YgRflKEZNrpLrQ` and `ChIJb0n5cWl3hlQRIbVGYLiTEgE` into constants with variable names that represent if they're valid or invalid placeIds?